### DISCUSSION TITLE
Why cannot change the camera icon

### DISCUSSION THREAD

**Discussion by 7230bayerngu** [MonwF/customiuizer#443] [General]
Xiaomi 14 pro, Hyper os 29, setting as the pictures 1-3![IMG_20240223_001350.jpg](https://github.com/MonwF/customiuizer/assets/160399165/40e63c13-4bf0-46bd-987c-151b9c28ad64)

![IMG_20240223_001427.jpg](https://github.com/MonwF/customiuizer/assets/160399165/b46237b9-91d2-4d2d-adf1-716e826f578a)

![IMG_20240223_001520.jpg](https://github.com/MonwF/customiuizer/assets/160399165/d74d25ca-9f40-4b0c-9dea-851161e64afe)

![IMG_20240223_001554.jpg](https://github.com/MonwF/customiuizer/assets/160399165/4d4aacfc-b920-45fe-b735-6abd0a020fca)

, cannot change the camera like picture 4

**Comment by MonwF** [comment]
设定了动作会自动修改图标

**Reply by 7230bayerngu** [reply]
具体怎么操作的啦？

**Reply by 7230bayerngu** [reply]
右滑会有相机图标出来，但是马上又自动改回齿轮形状的了

**Reply by MonwF** [reply]
你修改了右侧图标的长按动作，默认就会把相机图标替换掉

**Reply by 7230bayerngu** [reply]
感谢感谢

**Reply by 7230bayerngu** [reply]
你好，最近锁屏界面的手电筒失效了，这是什么情况的啦？
